.github/ISSUE_TEMPLATE/new-feature.yaml

```
@@ -0,0 +1,44 @@
+name: Request documentation for an upcoming product change
+description: Describe an upcoming change to Astro or Astronomer Software that needs documentation
```

```suggestion
description: Describe an upcoming change to Astro or Astronomer Software that needs documentation
```

I'm confused. Haven't you already created this template above?